schemas/com.roku/video/jsonschema/1-0-0

```diff
@@ -0,0 +1,115 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Context Schema for a Roku video player event",
+  "self": {
+    "format": "jsonschema",
+    "name": "video",
+    "vendor": "com.roku",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "videoId": {
+      "type": "string",
+      "description": "ID generated when video tracking of the video node was initialized.",
+      "maxLength": 255
+    },
+    "contentId": {
+      "type": [
+        "string",
+        "null"
+      ],
+      "description": "Played video content ID.",
+      "maxLength": 255
+    },
+    "contentTitle": {
+      "description": "Title of the played content.",
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 65535
+    },
+    "contentType": {
+      "description": "Type of the played content.",
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 255
+    },
+    "streamFormat": {
+      "description": "Type of content (e.g., mp4, wma, mkv).",
+      "type": [
+        "string",
+        "null"
+      ],
+      "maxLength": 255
+    },
```

Comment:
Whats the difference between these two properties? Can't figure it out from the descrition of `contentType`.

Reply:
This is really confusing, the official documentation isn't very helpful (the descriptions I used were from there) so it's up to interpretation. This is how I interpret it (your container vs codec explanation was close):

- streamFormat is the container format for the video. It comes from content metadata that is passed to the player during initialization.
- videoFormat is the codec of the played stream. It is automatically inferred by the video player.
- contentType is the category of played content. It can be one of audio, episode, movie, season, series. It is passed in the content metadata during video initialization.

I tried to update the descriptions a bit, please let me know if it is understandable now.

Reply:
Looks good! The examples help.